packages/style-engine/class-wp-style-engine-css-rules-store.php

```diff
@@ -47,6 +47,16 @@ public static function get_store( $store_name = 'default' ) {
 		}
 		return static::$stores[ $store_name ];
 	}
+
+	/**
+	 * Get an array of all available stores.
+	 *
+	 * @return WP_Style_Engine_CSS_Rules_Store[]
+	 */
+	public static function get_stores() {
```

Comment:
A `get_stores` in the Store class makes sense in order to get all registered/available stores. In all the other PRs I made, this one was always a requirement so I'm adding it here too as it makes it easier to abstract the logic for `$stores` in the `WP_Style_Engine` object as well.